snuba/subscriptions/executor_consumer.py

```diff
@@ -495,3 +495,8 @@ def join(self, timeout: Optional[float] = None) -> None:
             logger.debug("Flushing executor producer")
             messages = producer.flush(*[remaining] if remaining is not None else [])
             logger.debug(f"{messages} executor messages pending delivery")
+
+        # Flush querylog producer
+        logger.debug("Flushing querylog producer")
+        state.flush_producer()
```

Comment:
would it be worth having metrics on how long it's taking to flush the querylog producer? cause now we will be flushing on rebalances as well right?

Reply:
Yeah, you're right. I actually don't really want to flush it on rebalance. Let me see if I can move this somewhere else so that doesn't happen.

Reply:
Moved it back out of the strategy, let me know what you think